src/app/exam/exam-add/exam-add.component.html

```diff
@@ -0,0 +1,46 @@
+<h2 align="center">Add a new Exam</h2>
+
+<div class="jumbotron">
+  <h4 class="display-4">New Exam</h4>
+  <p class="lead">Please, provide your theory</p>
+
+  <form id="user-form" (ngSubmit)="onSubmit()" #form="ngForm">
+    <fieldset>
+      <div id="formulaire">
+
+        <!-- name input -->
+        <div class="form-group" [class.was-validated]="name.dirty || name.touched">
+          <label class="control-label" for="name">Name*</label>
+          <input id="name" name="name" type="text" class="form-control"
+                 [(ngModel)]="exam.name" #name="ngModel" required>
+          <div class="invalid-feedback">Name field is required</div>
+        </div>
+
+        <!-- question input -->
+        <div class="form-group" [class.was-validated]="question.dirty || question.touched">
+          <label class="control-label" for="questionId">Questions:*</label>
+          <select id="questionId" name="questionId" type="text" class="form-control"
+                  [(ngModel)]="exam.question" #question="ngModel" required multiple>
+            <option *ngFor="let question of questions" [value]="question.id" >{{question.statement}}</option>
```

Comment:
The backend expects the questions array to contain the URIs of the selected Questions. This can be achieved using:
`[value]="question._links.self.href"`